qdev_wrappers/alazar_controllers/acquisition_parameters.py

```diff
@@ -37,10 +37,9 @@ def set_raw(self, value):
             value: value to set the parameter to
         """
         self._check_and_update_instr(value, param_name=self.name)
-        self._save_val(value)
 
     def get_raw(self):
```

Comment:
is anybody using [`aquisition_parametersold`](https://github.com/qdev-dk/qdev-wrappers/blob/3412bc01e13060ea88428a876477700d9e0e5efb/qdev_wrappers/alazar_controllers/acquisition_parametersold.py)? those also contain some `_save_val` calls. Perhaps that module can just be removed?